tracer/src/Datadog.Trace/Configuration/IExtraServicesProvider.cs

```diff
@@ -0,0 +1,13 @@
+﻿// <copyright file="IExtraServicesProvider.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+namespace Datadog.Trace.Configuration;
+
+internal interface IExtraServicesProvider
```

Comment:
nit: If you're never mocking this or creating other versions, then not much point in adding an interface

Reply:
Removed in 14f828b6ca89a2f776967e35a647b6493ec40c97